README.md

```diff
@@ -66,11 +78,17 @@ The migrator will look for a `bootstrap.cql` file for setting up the keyspace.
 ## Standalone usage
 
 ```sh
-$ java -Dhosts=localhost,192.168.1.1 -Dkeyspace=my_keyspace -Ddirectories=cql-common,cql-local -jar cqlmigrate.jar
+$ java -Dhosts=localhost,192.168.1.1 -Dport=9042 -DlocalDC=DC1 -Dkeyspace=my_keyspace -Ddirectories=cql-common,cql-local -jar cqlmigrate.jar
 ```
 
 Specify credentials, if required, using `-Dusername=<username>` and `-Dpassword=<password>`.
 
+Specify optional properties, if required, using
+* `-Dprecheck=<true/false>` default `false` 
+* `-DtableCheckerTimeout=<duration>` default no wait
```

Comment:
should we mention that this feature ONLY works for AWS keyspaces?

Reply:
I mentioned it in the section "What it does". I can put it in bracket here as well.